src/Primitives/Container.purs

```diff
@@ -210,20 +216,30 @@ component =
           then pure a
           else a <$ case ms of
             Down -> H.modify $ seeks (_ { mouseDown = true })
-            Up   -> H.modify $ seeks (_ { mouseDown = false })
+            Up   -> do
+              H.modify $ seeks (_ { mouseDown = false })
+              H.raise ContainerClicked
 
       Blur a -> do
         (Tuple _ st) <- getState
         if not st.open || st.mouseDown
           then pure a
-          else a <$ (eval $ Visibility Off a)
+          else a <$ (eval $ SetVisibility Off a)
```

Comment:
I like this pattern of triggering eval queries rather than duplicate logic, even if simple. 👍